onnx/__init__.py

```diff
@@ -134,7 +134,7 @@
 # Supported model formats that can be loaded from and saved to
 # The literals are formats with built-in support. But we also allow users to
 # register their own formats. So we allow str as well.
-_SupportedFormat = Union[Literal["protobuf", "textproto"], str]
+_SupportedFormat = Union[Literal["protobuf", "textproto", "onnxtxt", "json"], str]
```

Comment:
so, the format has a unique name "onnxtxt", but corresponding extensions can be several different names?

Reply:
That's right